packages/contracts/deploy/update/to_v1.3.0/11_TokenVoting_Plugin_conclude.ts

```diff
@@ -0,0 +1,11 @@
+import {DeployFunction} from 'hardhat-deploy/types';
+import {HardhatRuntimeEnvironment} from 'hardhat/types';
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  console.log('\nConcluding TokenVoting Plugin Update');
+
+  hre.aragonToVerifyContracts.push(
+    await hre.deployments.get('TokenVotingSetup')
```

Comment:
are we sure `TokenVotingSetup` is the new build and not the old one?
i would rather use a diferent name as mentioned in my previous comment, or write a test to be sure